### ISSUE TITLE
[embree3:x64-osx] build failure

### ISSUE THREAD

**Issue by neil3d** [microsoft/vcpkg#15356]
**Host Environment**
- OS: [MacOS Big Sur, 11.1]
- Compiler: AppleClang 12.0.0.12000032

**To Reproduce**
`./vcpkg install embree3`

**Failure logs** 
- vcpkg/buildtrees/embree3/config-x64-osx-dbg-err.log
```
CMake Error at /Users/neil3d/github/vcpkg/scripts/buildsystems/vcpkg.cmake:634 (_find_package):
  Found package configuration file:

    /Users/neil3d/github/vcpkg/installed/x64-osx/share/tbb/TBBConfig.cmake

  but it set TBB_FOUND to FALSE so package "TBB" is considered to be NOT
  FOUND.
Call Stack (most recent call first):
  common/tasking/CMakeLists.txt:13 (FIND_PACKAGE)
```

- vcpkg/buildtrees/embree3/config-x64-osx-dbg-out.log
```
-- The C compiler identification is AppleClang 12.0.0.12000032
-- The CXX compiler identification is AppleClang 12.0.0.12000032
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Missed required Intel TBB component: tbb
--   one or both of:
   /Users/neil3d/github/vcpkg/installed/x64-osx/lib/libtbb.a
    /Users/neil3d/github/vcpkg/installed/x64-osx/debug/lib/libtbb_debug.a
   files must exist.
-- Configuring incomplete, errors occurred!
See also "/Users/neil3d/github/vcpkg/buildtrees/embree3/x64-osx-dbg/CMakeFiles/CMakeOutput.log".
See also "/Users/neil3d/github/vcpkg/buildtrees/embree3/x64-osx-dbg/CMakeFiles/CMakeError.log".
```
**Additional context**
I make sure that TBB is installed:
```
./vcpkg install tbb
Computing installation plan...
The following packages are already installed:
    tbb[core]:x64-osx -> 2020_U3#2
Package tbb:x64-osx is already installed
```

**Comment by NancyLi1013** [comment]
Hi @neil3d 

Thanks for posting this issue.

Could you please update

` FIND_PACKAGE(TBB REQUIRED tbb)`

as

` FIND_PACKAGE(TBB CONFIG REQUIRED tbb)`?

https://github.com/embree/embree/blob/master/common/tasking/CMakeLists.txt#L13

**Comment by NancyLi1013** [comment]
Hi @neil3d 

Thanks for posting this issue.

Could you please update

` FIND_PACKAGE(TBB REQUIRED tbb)`

as

` FIND_PACKAGE(TBB CONFIG REQUIRED tbb)`?

https://github.com/embree/embree/blob/master/common/tasking/CMakeLists.txt#L13

**Comment by NancyLi1013** [comment]
It seems that there are no debug libs generated. But I am not sure if it is related with this.

I can build `embree3 `successfully on my machine.

**Comment by JackBoosY** [comment]
Fixed in #17925.
